tutorial/4_hextof_workflow.ipynb

```diff
@@ -108,27 +322,28 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "plt.figure()\n",
-    "res.plot.line(x='dldTime')"
+    "We binned not only in `dldTime` but also in `sampleBias`. This allows us to separate the spectra obtained at different bias values. "
```

Comment:
I suggest to add a cell or two explaining the overview histogram function here (view_event_histogram). With that you can easily inspect the values of sampleBias sampled, and the dldTime range of interest.

Reply:
I agree and will add this. Also, I never tried it and I better know how it works before the BT

Reply:
I added respective cells